index.html

```diff
@@ -2487,5 +2487,33 @@ <h3>Selective Disclosure</h3>
 
     </section>
 
+    <section>
+      <h2>Internationalization Considerations</h2>
```

Comment:
The I18N WG generally does not request the creation of "Internationalization Considerations" sections unless there is a really specific message that needs to be communicated. We generally prefer that linguistic, cultural, regional, and other considerations be dealt with in the section(s) that are applicable (and where guidance will most likely be seen by implementers). 

I would be much happier if you put a note block into the "representing time" section (the original one linked to but the new "representing time" sub-section in this PR) that says something like:

```html
<!-- note this links to a term in the i18n-glossary -->
<div class="note">
<p>When displaying date and time values related to the
    validity of cryptographic proofs, implementers should 
    respect the <a>locale</a> and possibly local calendar
    preferences of the user. 
    Conversion of timestamps to local time values should
    consider the time zone expectations of the user.</p>
</div>
```

Reply:
So we could just put your recommended text alongside the **created** parameter in the  **Proofs** (section 2.1) which is the only place we have a date/time.

Reply:
@Wind4Greg exactly so.

Note that I18N in in the process of doing a review and there is a `pending` issue related to this (with different textual recommendations in it) [here](https://github.com/w3c/i18n-activity/issues/1729)

Reply:
Thanks @aphillips we'll want to be consistent with the VC Data Model 2.0 and I particularly like your point about handling cases with/without timezone information!

Reply:
Ok, I'll make the change as you suggest @aphillips -- thank you. I'll also try to cross-link to the vc-data-model section on time representation so we take advantage of the text over there as well.

Reply:
Fixed in bae11c651e9964ff1f8c7dc585aa65cc54bce0d9.